README.md

```diff
@@ -41,7 +41,7 @@ NOTE: Be aware that the configuration in debian/rules expects tabs instead of sp
 
 Once the package is built, you have a virtualenv contained in a Debian
 package and upon installation it gets placed, by default, under
-`/usr/share/python/<packagename>`.
+`/opt/venvs/<packagename>`.
```

Comment:
Take a look at the docs change that are part of this commit, specifically the things related to `DH_VIRTUALENV_INSTALL_ROOT`. That'll allow you to relocate an installation to `/usr/share` for example or anywhere else you would like 😄.